fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -165,6 +159,12 @@ protected boolean shouldCacheProperties() {
         return extraPropertiesStore() != null && (!isReadonly() || this.propertiesDirectory != null);
     }
 
+
+    @Override
+    public boolean isRoot(final String id) {
```

Comment:
Why is providing a pass-through implementation of `isRoot()` and `isContentNode()` required here?